### PR TITLE
also expanded to DTG

### PR DESCRIPTION
**Pull request by james-trayford**
@EvgeniiChaikin to be merged into `multi-z-scripts` prior to merging https://github.com/SWIFTSIM/velociraptor-comparison-data/pull/185

### CODE REVIEW

File: data/GalaxyMetallicityDusttoGasRatio/conversion/convertKonstantopoulou23.py [hunk 1]
```diff
@@ -82,34 +70,43 @@ def read_tex_tables():
...
+    multi_z = MultiRedshiftObservationalData()
 
     comment = (
         "Konstantopoulou et al. (2023). Obtained using VLT, X-shooter & HST for a combination of"
         "sources: MW, LMC, SMC, QSO-DLA and GRBs."
     )
     citation = "Konstantopoulou et al. (2023)"
     bibcode = "10.48550/arXiv.2310.07709"
-    name = "Dust-to-gas ratio as a function of metallicity (represented as O abundance)"
+    name = "Dust-to-metal ratio as a function of metallicity (represented as Oxygen abundance)"
```
Comment (EvgeniiChaikin) [inline data/GalaxyMetallicityDusttoGasRatio/conversion/convertKonstantopoulou23.py:82]: @james-trayford shouldn't it say `Dust-to-gas' in the comment?

Comment (james-trayford) [reply data/GalaxyMetallicityDusttoGasRatio/conversion/convertKonstantopoulou23.py]: Yes, missed that, thanks

**Comment by MatthieuSchaller** [comment]
Does this still need merging?

**Comment by EvgeniiChaikin** [comment]
I believe this needs to be rebased to the just-updated master branch and then merged.

**Comment by james-trayford** [comment]
I actually made the same fix as @EvgeniiChaikin in Peroux so the rebase did nothing - I wanted this merged into his branch before merging to the master, but I guess I can open this on the master instead now

**Pull request closed without merging**